main.go

```diff
@@ -23,15 +23,18 @@ Oh is released under an MIT license.
 package main
 
 import (
-	"github.com/michaelmacinnis/oh/pkg/cell"
-	"github.com/michaelmacinnis/oh/pkg/system"
-	"github.com/michaelmacinnis/oh/pkg/task"
-	"github.com/peterh/liner"
+	"flag"
+	"fmt"
 	"os"
 	"path"
 	"path/filepath"
 	"sort"
 	"strings"
+
```

Comment:
Looks like you need to run ```go fmt```.

Reply:
I see. I use  [acmego](https://github.com/9fans/go/tree/master/acme/acmego) which applies [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports). It formats the imports that way, and applies the same formatting as `go fmt` as per its documentation. If the convention is to use just go fmt, I'll do it that way and disable `goimports`, even though I think it is quite useful ;)

Reply:
Yeah, just ```go fmt```, please. Otherwise next time I touch those files it will look like a change to the imports.